src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import { Movie } from '../../types/movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+
+  <div className="movies">
+    {movies.map(movie => (
+      <div
+        className="card"
+        data-cy="card"
+        key={movie.imdbId}
+      >
+        <MovieCard {...movie} />
```

Comment:
It's not the best solution to pass props
If you'll have more that 100 props in future in movie object - they all will be passed to MovieCard
You need to pass all object as movie property in MovieCard props
And then get needed props from this object in MovieCard component